x-pack/plugins/apm/public/components/shared/EnvironmentFilter/index.tsx

```diff
@@ -30,29 +31,20 @@ function updateEnvironmentUrl(
 }
 
 const OptionInputContainer = styled.div`
-  min-width: 192px;
+  min-width: 128px;
 `;
 
-const EnvironmentLabel = styled.span`
-  color: #adb2b9;
-  margin-right: 6px;
+const EnvironmentFilterLabel = styled.div`
```

Comment:
@ogupte Don't think we should be opting to use a styled component, at least not with this many specific values that don't use the EUI styles. My intention was to use the `prepend` label that is available for the [form control](https://elastic.github.io/eui/#/forms/form-controls) elements, but I see that those props are not available for the `EuiSuperSelect`. I suppose we can request to be add `prepend` and `append` props to the `EuiSuperSelect` to make them look consistent in this matter, or do you think there's other ways of achieving it?

<img width="293" alt="Screenshot 2019-05-02 at 12 40 52" src="https://user-images.githubusercontent.com/4104278/57070595-42257e80-6cd8-11e9-9709-b185772195b9.png">

<img width="426" alt="Screenshot 2019-05-02 at 12 45 38" src="https://user-images.githubusercontent.com/4104278/57070601-45b90580-6cd8-11e9-829c-bddf078affd8.png">

Reply:
Codesandbox: https://codesandbox.io/s/92yxm71mqw

Reply:
I think `EuiSelect` should suffice in this case, because we don't have multi-select for environments and the selected state is actually showing in HTML element.